test/integration/util/util.go

```diff
@@ -33,6 +33,8 @@ import (
 	commonutil "k8s.io/minikube/pkg/util"
 )
 
+const kubectlBinary = "kubectl-v1.6.0-alpha.1"
```

Comment:
We shouldn't have to modify it with every release, as there should be compatibility between alpha and beta 1.6s, but we will want to upgrade kubectl to the official 1.6 kubectl once its released.